app/Console/Commands/CreateAdmin.php

```diff
@@ -20,13 +20,14 @@ class CreateAdmin extends Command
      * @property string $password
      * @property boolean $activated
      * @property boolean $show_in_list
+     * @property boolean $autoassign_licenses
      * @property \Illuminate\Support\Carbon|null $created_at
      * @property mixed $created_by
      */
 
 
 
-    protected $signature = 'snipeit:create-admin {--first_name=} {--last_name=}  {--email=}  {--username=}  {--password=}   {show_in_list?}';
+    protected $signature = 'snipeit:create-admin {--first_name=} {--last_name=}  {--email=}  {--username=}  {--password=} {show_in_list?} {autoassign_licenses?}';
```

Comment:
So as to make the command nicer, I would think you might want to have the `{show_in_list?}` and `{autoassign_licenses?}` arguments into options - `{--show_in_list}` and `{--autoassign_licenses}`.

This way, the command line you would type would be:

```sh
php artisan snipeit:create-admin --first_name=Brady --last_name=Bunch --email=something@example.com \
 --username=brady --password=bradysux --show_in_list --autoassign_licenses
```

rather than:

```sh
php artisan snipeit:create-admin --first_name=Brady --last_name=Bunch --email=something@example.com \
 --username=brady --password=bradysux true true
```

But, that would be a backwards-compatibility break, so I could see why you might not want to do it.